app/javascript/app/pages/regions/regions-ghg-emissions/regions-ghg-emissions-component.jsx

```diff
@@ -1,21 +1,161 @@
+
+  renderDropdown(field, multi) {
+    const { selectedOptions, filterOptions, t } = this.props;
+    const value = selectedOptions && selectedOptions[field];
+    const options = filterOptions[field] || [];
+
+    const label = t(
+      `pages.regions.regions-ghg-emissions.labels.${kebabCase(field)}`
+    );
+
+    if (multi) {
+      const values = castArray(value).filter(v => v);
+
+      return (
+        <Multiselect
+          key={field}
+          label={label}
+          options={options}
+          onValueChange={selected => this.handleFilterChange(field, selected)}
+          values={values}
+          theme={{ wrapper: dropdownStyles.select }}
+          hideResetButton
+        />
+      );
+    }
+    return (
+      <Dropdown
+        key={field}
+        label={label}
+        options={options}
+        onValueChange={selected => this.handleFilterChange(field, selected)}
+        value={value || null}
+        theme={{ select: dropdownStyles.select }}
+        hideResetButton
+      />
+    );
+  }
+
+  renderChart() {
+    const { chartData, onYearChange } = this.props;
+
+    if (!chartData || !chartData.data) return null;
+
+    return (
+      <Chart
+        theme={{
+          legend: styles.legend,
+          projectedLegend: styles.projectedLegend
+        }}
+        type="line"
+        config={chartData.config}
+        data={chartData.data}
+        projectedData={chartData.projectedData || []}
+        domain={chartData.domain}
+        dataOptions={chartData.dataOptions}
+        dataSelected={chartData.dataSelected}
+        height={500}
+        loading={chartData.loading}
+        getCustomYLabelFormat={value => format('.3s')(value)}
+        showUnit
+        onLegendChange={v => this.handleFilterChange('sector', v)}
+        onMouseMove={onYearChange}
+      />
+    );
+  }
+
   render() {
+    const { emissionParams, selectedYear, t } = this.props;
+
     return (
-      <React.Fragment>
-        <p className={styles.text}>The RegionsGhgEmissions section</p>
-        <ClimatePlansProvider />
-      </React.Fragment>
+      <div className={styles.page}>
+        <SectionTitle
+          title={t('pages.regions.regions-ghg-emissions.title')}
+          description={t('pages.regions.regions-ghg-emissions.description')}
+        />
+        <div className={styles.chartMapContainer}>
+          <div className={styles.filtersChartContainer}>
+            <div className={styles.dropdowns}>
+              {this.renderDropdown('sector', true)}
+              {this.renderDropdown('gas', true)}
+              {this.renderDropdown('metric', false)}
+              <InfoDownloadToolbox
+                className={{ buttonWrapper: styles.buttonWrapper }}
+                slugs=""
+                downloadUri=""
+              />
+            </div>
+            <div className={styles.chartContainer}>
+              {this.renderChart()}
+            </div>
+          </div>
+          <GHGMap selectedYear={selectedYear} />
```

Comment:
Could we avoid rendering this on mobile using  the TabletLandscape component? I know we are not taking responsive into account yet. But it will be a fast change:

Before:
![image](https://user-images.githubusercontent.com/9701591/51383392-9f2a6980-1b19-11e9-9b04-361db5f792ec.png)

After: 
![image](https://user-images.githubusercontent.com/9701591/51383402-a5204a80-1b19-11e9-9e0d-d8319aff5f98.png)

Reply:
:+1: